framework/src/wifi_manager/wifi_manager.c

```diff
@@ -629,7 +629,7 @@ wifi_manager_result_e _handler_on_uninitialized_state(_wifimgr_msg_s *msg)
 
 	/* update wifi_manager info by being protected by w_info_mutex */
```

Comment:
Where is w_info_mutex?
You need to fix or remove the comment.

Reply:
ok, I will fix